Python/generated_cases.c.h

```diff
@@ -1110,7 +1107,14 @@
                 assert(PyStackRef_IsNull(null));
                 assert(Py_TYPE(callable_o) == &PyMethod_Type);
                 self = PyStackRef_FromPyObjectNew(((PyMethodObject *)callable_o)->im_self);
+                #ifdef Py_GIL_DISABLED /* flush specials */
+                stack_pointer[-1 - oparg] = self;
+                #endif /* flush specials */
                 method = PyStackRef_FromPyObjectNew(((PyMethodObject *)callable_o)->im_func);
+                #ifdef Py_GIL_DISABLED /* flush specials */
+                stack_pointer[-2 - oparg] = method;
+                #endif /* flush specials */
+                stack_pointer[-2 - oparg] = method;
```

Comment:
I updated the generator to avoid saving the same value twice. I got rid of the `#ifdef Py_GIL_DISABLED` guards because I think they would add too much complexity for no real gain.